app/models/concerns/audit_extensions.rb

```diff
@@ -101,6 +102,14 @@ def filter_encrypted
     end
   end
 
+  def filter_passwords
+    if action == 'update'
+      audited_changes['password'] = [N_('[redacted]'), N_('[redacted]')]
```

Comment:
perhaps extract the string to a constant?

Reply:
done, and unified it with the encrypted redaction as well.